src/components/sidebar/lineView/LineRoutesTab.tsx

```diff
@@ -0,0 +1,34 @@
+
+import React from 'react';
+import { inject, observer } from 'mobx-react';
+import { LineStore } from '~/stores/lineStore';
+import s from './lineRoutesTab.scss';
+
+interface ILineRoutesTabProps {
+    lineStore?: LineStore;
+}
+
+@inject('lineStore')
+@observer
+class LineRoutesTab extends React.Component<ILineRoutesTabProps>{
+
```

Comment:
Remove extra line break

Reply:
Removed. For me doesn't matter though if there is a line break or not